modules/API/external/external_api.js

```diff
@@ -121,6 +123,14 @@ function JitsiMeetExternalAPI(domain, room_name, width, height, parentNode,
     this.url = (noSsl) ? "http" : "https" +"://" + domain + "/";
     if(room_name)
         this.url += room_name;
+
+    if(jwt) {
+        this.url += '?jwt=' + jwt;
+        // Disable the ring overlay by default.
+        interfaceConfigOverwrite.disableRingOverlay
+            = interfaceConfigOverwrite.disableRingOverlay || true;
```

Comment:
Just a passing question: is there a case where disableRingOverlay is explicitly set to false? Would that then make it true when undesired?